### DISCUSSION TITLE
Poor man's auto-format

### DISCUSSION THREAD

**Discussion by dimas-cyriaco** [nvim-neorg/neorg#1159] [General]
For those how are looking for a simple auto-format for Neorg:

```lua
vim.api.nvim_create_autocmd({ "BufWritePre" }, {
  group = vim.api.nvim_create_augroup("norg_format", { clear = true }),
  pattern = { "*.norg" },
  command = 'exe \"normal gg=G\\<c-o>]\"',
})
```

It's working reasonably well in my testing.

**Comment by pysan3** [comment]
... until you face this issue: https://github.com/nvim-neorg/neorg/issues/1100

**Reply by dimas-cyriaco** [reply]
I have already faced it. But it's better than having to format manually.
